CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.0.0](https://github.com/paritytech/substrate-api-sidecar/compare/v1.0.0-rc4...v1.0.0) (2020-10-23)
+
+
+### ⚠ BREAKING CHANGES
+
+* Remove all v0 routes to prepare for v1 release
+
+### Features
+
+* Remove all v0 routes to prepare for v1 release ([410a2e9](https://github.com/paritytech/substrate-api-sidecar/commit/410a2e9251bf341b9e0f151bccf9c83617c7673f))
```

Comment:
I wouldn't regard remove code as a `feature`, I think mentioning it as a breaking change is sufficient.

Reply:
I agree, let's move this up to the breaking changes section.